CHANGELOG.md

```diff
@@ -6,12 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed 
+### Added
+- Added more memory allocation to E2E tests
```

Comment:
I don't think end users will care about this?

Reply:
I will remove it.